.dockerfiles/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.12-slim as builder
```

Comment:
N00b question: why slim? Just for space?

Reply:
Good question:

```
$ docker images | grep 3.12
python     3.12                 a3aef63c6c10   3 weeks ago         1.02GB
python     3.12-slim            7f98baf23246   3 weeks ago         130MB
```

It's **much** >800MB smaller which is great for this type of website since we don't need anything fancy.

Reply:
Yep, that's a huge difference. Thanks!